assets/data/team.json

```diff
@@ -1,4 +1,38 @@
 [
+<<<<<<< HEAD
```

Comment:
unwanted line

Reply:
yes it was due to merge conflicts I have resolved it